core/src/main/scala/akka/kafka/Subscriptions.scala

```diff
@@ -56,6 +72,15 @@ object Subscriptions {
       extends ManualSubscription {
     def withRebalanceListener(ref: ActorRef): Assignment =
       Assignment(tps, Some(ref))
+
+    override def toString: String = {
+      val listener = rebalanceListener match {
+        case Some(ref) => s",rebalanceListener=$ref"
+        case None => ""
+      }
+      s"Assignment(${tps.mkString(",")}$listener)"
```

Comment:
Parenthesis here and in other sunbscription definitions will be converted to the URL-encoded-format when used in the `name` attribute. Should we use something more URL friendly, like `/`?